src/overloads/gradient_tracer.jl

```diff
@@ -153,14 +153,31 @@ function generate_code_gradient_2_to_1_typed(
     is_der1_arg1_zero_g = is_der1_arg1_zero_global(f)
     is_der1_arg2_zero_g = is_der1_arg2_zero_global(f)
 
-    expr_tracer_type = quote
-        function $M.$fname(tx::$SCT.GradientTracer, ::$Z)
-            return @noinline $SCT.gradient_tracer_1_to_1(tx, $is_der1_arg1_zero_g)
+    if f === Base.:*  # special case
```

Comment:
Add a TODO note referring to https://github.com/adrhill/SparseConnectivityTracer.jl/issues/244.